iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/ResourceDeclarationImpl.java

```diff
@@ -141,7 +142,11 @@ public StringLiteral type() {
   @Override
   public List<Property> properties() {
     if (body.is(Kind.OBJECT_EXPRESSION)) {
-      return ((ObjectExpression) body).properties();
+      return ((ObjectExpression) body).properties().stream()
+        .filter(propertyTree -> "properties".equals(((TextTree) propertyTree.key()).value()))
+        .map(p -> (Collections.<Property>unmodifiableList(((ObjectExpression) ((Property) p).value()).properties())))
+        .findFirst()
+        .orElse(List.of());
```

Comment:
I believe this behavior should also be applied In case of `ForExpression` and `IfCondition`.
In both cases, the object body represents a resource, and so we should provide the `properties` field.

Example from Azure bicep documentation:
```
resource vmName_omsOnboarding 'Microsoft.Compute/virtualMachines/extensions@2023-03-01' = if (!empty(logAnalytics)) {
  name: '${vmName}/omsOnboarding'
  location: location
  properties: {
    publisher: 'Microsoft.EnterpriseCloud.Monitoring'
    type: 'MicrosoftMonitoringAgent'
    typeHandlerVersion: '1.0'
    autoUpgradeMinorVersion: true
    /* ... */
  }
}
```